### PR TITLE
New extension event to send message from event (currently disabled)

### PR DESCRIPTION
**Pull request by krisc**
@jeluard @StatusSceptre this PR is for the bounty #7354 . The original PR on my submission for the bounty wasn't building and sending a new PR made it simpler for me. Sorry for the clutter :P

### CODE REVIEW

File: env/dev/env/config.cljs [hunk 1]
```diff
@@ -1,6 +1,6 @@
 (ns env.config)
 
 (def figwheel-urls {:android "ws://localhost:3449/figwheel-ws",
```
Comment (jeluard) [inline env/dev/env/config.cljs]: Please remove this from the PR

**Review by jeluard — CHANGES_REQUESTED** [review]

**Review by flexsurfer — APPROVED** [review]

**Review by jeluard — APPROVED** [review]

**Comment by asemiankevich** [comment]
@krisc could you please provide an extension sample for this ?

**Comment by jeluard** [comment]
@krisc You will have to rebase too

**Comment by krisc** [comment]
> @krisc could you please provide an extension sample for this ?

@asemiankevich please see this comment https://github.com/status-im/status-react/pull/7560#issuecomment-468015731

Also, note that @flexsurfer and @jeluard decided to pull in this code but currently disabled until a better security model is figured out. So to use this, you'll have to uncomment the code first.

**Review by yenda — CHANGES_REQUESTED** [review]
For code that can be optionally enabled, we use environment variable. You can create one in `status-im.utils.config` and surround your code in a `when` close using the variable you create

**Comment by asemiankevich** [comment]
> > @krisc could you please provide an extension sample for this ?
> 
> @asemiankevich please see this comment [#7560 (comment)](https://github.com/status-im/status-react/pull/7560#issuecomment-468015731)

When trying to install this errors are shown and extension could not be installed:
image

@jeluard @flexsurfer any hint?

**Comment by jeluard** [comment]
@asemiankevich Yes this is expected, there is nothing to be tested here.
Actually it might make sense to not merge this PR, but still give the bounty. @StatusSceptre WDYT?

**Comment by asemiankevich** [comment]
> @asemiankevich Yes this is expected, there is nothing to be tested here.
> Actually it might make sense to not merge this PR, but still give the bounty. @StatusSceptre WDYT?

could you please add then this disappointing `skip-manual-qa` label just in case?

**Comment by krisc** [comment]
> @asemiankevich Yes this is expected, there is nothing to be tested here.
> Actually it might make sense to not merge this PR, but still give the bounty. @StatusSceptre WDYT?

@jeluard  You still think it's not worth merging with the optional enabling as @yenda suggested? In any case, I'll wait on what @StatusSceptre thinks before continuing on.

**Comment by krisc** [comment]
@StatusSceptre care to comment on this?

**Comment by StatusSceptre** [comment]
@jeluard @yenda are we decided that we do not merge this, but still payout the bounty? Am going to go ahead and do that as it is just for 80 DAI.

**Comment by jeluard** [comment]
@StatusSceptre Exactly!

**Pull request closed without merging**